scripts/create_type_file.py

```diff
@@ -29,11 +29,16 @@ def compare_type_files_for_all_networks(chains_file):
         index += 1
         print(
             f"Generating has started for: {chain['name']}. {index}/{len(chains_file)}")
-        if chain['name'] in ['Moonbeam', 'Moonriver', 'Moonbase', 'Crab', 'Ethereum']:
-            # TODO need to implement creation type file for EVM networks
-            print(
-                f"Temporary can't generate type files for EVM networks, {chain['name']} was skipped")
-            continue
+        chain_options = chain.get('options')
+        skip_options = ['ethereumBased', 'noSubstrateRuntime']
+        
+        if chain_options is not None:
+            need_skip = [option for option in skip_options if option in chain_options]
+            if need_skip:
+                # TODO need to implement creation type file for EVM networks
+                print(f"Temporary can't generate type files for EVM networks, {chain['name']} was skipped")
```

Comment:
Does it mean Moonbeam and Moonriver won't be updated? Are there some specific problems with them?

Reply:
Yes, right, types for EVM networks won't be updated, flow which will update this kind of types have not implemented yet.